<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoConfigurationPage</class>
 <widget class="QWidget" name="DemoConfigurationPage">
  <property name="windowTitle">
   <string>Demo server</string>
  </property>
  <property name="windowIcon">
   <iconset resource="demo.qrc">
    <normaloff>:/demo/window-duplicate.png</normaloff>:/demo/window-duplicate.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tunables</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
      <item row="1" column="1">
       <widget class="QSpinBox" name="framebufferUpdateInterval">
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Key frame interval</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Memory limit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="multithreadingEnabled">
        <property name="text">
         <string>Use multithreading (experimental)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="memoryLimit">
        <property name="suffix">
         <string> MB</string>
        </property>
        <property name="minimum">
         <number>64</number>
        </property>
        <property name="maximum">
         <number>512</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Update interval</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="keyFrameInterval">
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections/>
</ui>
